src/Parse/Ini.php

```diff
@@ -121,7 +121,7 @@ public function expandProperty(&$array, $key, $value)
             return $array = $value;
         }
 
-        $keys = explode('|', $key);
+        $keys = explode('|', (string) $key);
```

Comment:
When would this not be a string?

Reply:
Not sure what's going on here but it fails the `testArray` - the array keys?
again see comment below this is just a related to strict types.

screenshot

![2](https://user-images.githubusercontent.com/4506434/92624275-ef28e780-f28c-11ea-90c2-cf9d9e2fcc29.png)